packages/ui/atomic/create-atomic-component/package.json

```diff
@@ -15,8 +15,11 @@
     "release:phase1": "npx -p=@coveord/release npm-publish",
     "test": "node --experimental-vm-modules --no-warnings ../../../../node_modules/jest/bin/jest.js"
   },
-  "keywords": [],
-  "author": "",
+  "keywords": [
+    "coveo",
```

Comment:
add `cli` in addition to or `coveo-cli` instead of `coveo` ?

Reply:
```suggestion
    "coveo",
    "cli",
```